x-pack/plugins/lens/public/shared_components/toolbar_popover.scss

```diff
@@ -1,3 +1,3 @@
 .lnsVisToolbar__popover {
-  width: 365px;
```

Comment:
It looks like the added 20px is still not enough to prevent the overflow, as shown in the screenshot below. In my quick local testing, this needs to be at least 404px wide or greater to prevent the overflow.

![image](https://user-images.githubusercontent.com/3884767/156589786-95794020-5f3b-44d7-b0cc-08a7213f8805.png)